CoreFoundation/URL.subproj/CFURLSessionInterface.c

```diff
@@ -111,6 +111,14 @@ CFURLSessionEasyCode CFURLSession_easy_setopt_tc(CFURLSessionEasyHandle _Nonnull
     return MakeEasyCode(curl_easy_setopt(curl,  option.value, a));
 }
 
+CFURLSessionEasyCode CFURLSession_easy_setopt_blob(CFURLSessionEasyHandle _Nonnull curl, CFURLSessionOption option, void* data, size_t len) {
```

Comment:
Let's mark this `void *_Nonnull data` and only call it with non-nil data to silence a compiler warning